ftw/subsite/browser/sitemap.py

```diff
@@ -23,6 +23,8 @@ def siteMap(self):
 
         queryBuilder = SubsiteSitemapQueryBuilder(context)
         query = queryBuilder()
+        temp_types_completion = {'portal_type': ('Image', 'File', 'Link', 'News Item', 'Folder', 'Document', 'Event', 'ftw.simplelayout.ContentPage', 'ftw.subsite.Subsite')}
```

Comment:
@maethu could you please look at line 26, to help me find out why `ftw.subsite.Subsite` is not in the query anymore in plone 5.1 (updating the dict should not be the solution) 😃